test/test_utils/eks.py

```diff
@@ -273,7 +273,7 @@ def eks_forward_port_between_host_and_container(selector_name, host_port, contai
     )
 
 
-@retry(stop_max_attempt_number=20, wait_fixed=30000, retry_on_exception=retry_if_value_error)
+@retry(stop_max_attempt_number=30, wait_fixed=30000, retry_on_exception=retry_if_value_error)
```

Comment:
the worker node was not ready in 20 retrys. it will cause the test fail. increase to 30 to get more time to initialize a worker node.

Reply:
The worker node initialization took more than 10 minutes? This may be something that warrants a closer look. Sure, let's increase the number of tries to 30, but let's also dive deeper to root-cause this.